roles/repository/tasks/repository-Debian.yml

```diff
@@ -18,13 +18,14 @@
   ansible.builtin.stat:
     path: "{{ repository_key_files_directory }}"
   register: result
+  delegate_to: "{{ groups['manager'][0] }}"
```

Comment:
I trink it is better to delegate this one to localhost.

Reply:
yes, done in new commit